package.json

```diff
@@ -13,7 +13,9 @@
   },
   "scripts": {
     "start": "nf start",
-    "webdriver": "webdriver-manager update && webdriver-manager start"
+    "webdriver": "webdriver-manager update && webdriver-manager start",
+    "mochatests": "jest --watchAll",
+    "mochatests-with-coverage": "jest --watchAll --coverage"
```

Comment:
Can these be in grunt? Unless we're migrating away from grunt (which we could look into, for sure, I have no particularly attachment) it seems weird to have some testing commands in grunt and some as node scripts.